### ISSUE TITLE
Opacity

### ISSUE THREAD

**Issue by zxdawn** [deeplook/svglib#109]
Hi @claudep 
My svg's opacity is 0.2. But, the opacity is 1 after converted to pdf by svg2pdf.
This is the [url](https://drive.google.com/open?id=1c-D4c-GRVa7aghhm6yEeCiUKBB5sX9mZ) of svg file.

1. svg in Photoshop
![svg](https://user-images.githubusercontent.com/30388627/36931221-5f8b710e-1eec-11e8-9e47-3e8ff95a6116.png)

2. svg2pdf
![image](https://user-images.githubusercontent.com/30388627/36931230-8dd2e2b8-1eec-11e8-84c0-60842dbf6c1e.png)

**Comment by claudep** [comment]
You're right, svglib implements 'fill-opacity' and 'stroke-opacity', not the plain 'opacity' yet.

**Comment by zxdawn** [comment]
@claudep Thank you for reply~

**Comment by claudep** [comment]
You shouldn't close the issue, we should fix it first!

**Comment by zxdawn** [comment]
@claudep  Sorry. Reopen now.
